utils/args.py

```diff
@@ -0,0 +1,74 @@
+import argparse
+import os
+from data import Data
+import random
+import numpy as np
+import torch
+from torch.backends import cudnn
+
+
+class Config:
+
+    def __init__(self, args):
+        self.data = args.data
+        self.batch_size = args.batch_size
+        self.num_workers = args.num_workers
+        self.epochs = args.epochs
+        self.lr = args.lr
+        self.momentum = args.momentum
+        self.weight_decay = args.weight_decay
+        self.seed = args.seed
+        self.train = args.train
+        self.model = args.model
+        self.save_each_model = args.save_each_model
+        self.eval_period = args.eval_period
+        self.run_id = args.run_id
+
+
+def parse_args() -> Config:
+    parser = argparse.ArgumentParser(description="Visualizing Convolutional Neural Networks")
+
+    parser.add_argument("--data", type=Data, default=int,
+                        help="Dataset (1) MNIST, (2) CIFAR10, (3) CIFAR100, (4) IMAGENET")
+    parser.add_argument("--batch_size", type=int, default=128, help="Batch size")
```

Comment:
@SeskoDk i just added some arguments i thought might become handy. default values and overall values are not fixed yet. we should definitely talk about what we need / want.

Reply:
I totally agree.

Reply:
IMO, this awaits #1 because then i can also add the normalizer mode / type